accord-core/src/main/java/accord/coordinate/Coordinate.java

```diff
@@ -72,7 +72,8 @@ private void start()
     {
         // TODO (desired, efficiency): consider sending only to electorate of most recent topology (as only these PreAccept votes matter)
         // note that we must send to all replicas of old topology, as electorate may not be reachable
-        node.send(tracker.nodes(), to -> new PreAccept(to, tracker.topologies(), txnId, txn, route), this);
+        node.send(tracker.nodes(), to -> new PreAccept(to, tracker.topologies(), txnId, txn, route),
+                  node.commandStores().from(route), this);
```

Comment:
memoize? no need to recompute.

also, should probably use home key by default to pick?

Reply:
> memoize? no need to recompute.

`node.commandStores().from(route)` is called outside of the loop, so it is computed once.  Or do you mean do this in the constructor in case `start` is called multiple times?

Reply:
oh, my mistake. it looked like it was called within the continuation

Reply:
> also, should probably use home key by default to pick?

Pushed the change and removed one of the `from` methods as it was only used by `Routes`